torch/distributed/_sharded_tensor/ops/elementwise_ops.py

```diff
@@ -33,3 +30,7 @@ def gelu(types, args=(), kwargs=None, pg=None):
         # Manually set sharding_spec
         new_st._sharding_spec = copy.deepcopy(input._sharding_spec)
         return new_st
+
+
+gelu = register_elementwise_op(torch.nn.functional.gelu)
+relu = register_elementwise_op(torch.nn.functional.relu)
```

Comment:
nit: Don't need `gelu = register_elementwise_op(torch.nn.functional.gelu)`, you can simply do `register_elementwise_op(torch.nn.functional.gelu)`?

Reply:
That works, thanks!